src/templates/NavBar.jsx

```diff
@@ -15,9 +15,10 @@ const NavDropdownSection = ({ children, link }) => (
 
 const TemplateNavBar = ({ links, collectionInfo, resources }) => (
   <nav className="navbar is-transparent flex-fill">
-    <div className="bp-container">
+    <div className="bp-container" style={{ display: "flex" }}>
```

Comment:
Hmm, does the flex itself suffice? Just noting also that we deviate from using `classNames` here, but I'm unsure of how important consistency is here.

Reply:
Yeah I opted to deviate slightly away from the template here - the reason is that `flex` is provided by a parent component in the template which doesn't exist in the partial version we have in the preview, so I think this is the closest approximation to that

Reply:
what i meant was that we could instead define a css class that provides flex and give the classname as `bp-container flex`

Reply:
Ah right, changed in https://github.com/isomerpages/isomercms-frontend/pull/1466/commits/5e95895ede0aaba087a61b171a48876b3b71b6f7